.circleci/config.yml

```diff
@@ -192,16 +172,15 @@ jobs:
             if [ "$CIRCLE_BRANCH" = "master" -a -z "$CIRCLE_PULL_REQUEST" ]; then
               docker save nipype/nipype:base \
                           nipype/nipype:latest \
-                          nipype/nipype:py27 \
-                          nipype/nipype:py36 | gzip -1 > /tmp/docker/nipype-base-latest-py36-py27.tar.gz \
-              && du -h /tmp/docker/nipype-base-latest-py36-py27.tar.gz
+                          nipype/nipype:py36 | gzip -1 > /tmp/docker/nipype-base-latest-py36.tar.gz \
+              && du -h /tmp/docker/nipype-base-latest-py36.tar.gz
             fi
       - persist_to_workspace:
           root: /tmp
           paths:
             - docker
 
-  test_py3_fmri_fsl_spm:
+  test_fmri_fsl_spm:
```

Comment:
should we consider breaking off the workflows and workflow tests to niflows, before we break off 2.0?

Reply:
i'm also fine with making all the breaks with 2.0.

Reply:
I'm happy to start moving things to niflows. Have we figured out a working plan for that? Maybe could do a chat sometime this week to iron out a rough plan?

Reply:
i'll coordinate a time.